app/controllers/groups_controller.rb

```diff
@@ -0,0 +1,85 @@
+class GroupsController < ApplicationController
+  def index
+    # currently also using this endpoint to pull down suggested groups
+    groups = if params[:limit].present? # Get a specific amount of trending
+      Group.trending(params[:limit].to_i)
+    elsif params[:trending].present? # Get all trending groups, paginated
+      Group.trending(0).page(params[:page]).per(20)
```

Comment:
If that's not a valid assumption then `TrendingAnime.list` is broken and so is `TrendingGroups.list` by proxy.  How can we fulfill that assumption if it's false?